www/templates/diary/place_list_item.html

```diff
@@ -2,23 +2,23 @@
     <!-- Place Item Template  -->
     <ion-item class="unified-diary-item" style="background-color: transparent;" class="list-item">
         <!-- Start Time Tag -->
-        <!-- <div class="start-time-tag">{{trip.display_end_time}}</div> -->
+        <!-- <div class="start-time-tag">{{place.display_enter_time}}</div> -->
         <div class="diary-card short" style="padding-top: 10px;">
             <div style="width: 100%; margin: auto; padding: 5px">
                 <div class="row place-title">
                     <!-- Place (Destination of Prev. Trip) -->
                     <i class="icon ion-location"></i>
-                    <b>{{trip.end_display_name}}</b>
+                    <b>{{place.display_name}}</b>
                 </div>
                 <div style="margin: 10px">
-                    <enketo-add-note-button notes-config="configPlaceNotes" surveys="surveys" trip="trip"
-                        time-bounds="timeBounds" dataKey="manual/place_addition_input" style="margin: auto">
+                    <enketo-add-note-button notes-config="configPlaceNotes" timeline-entry="place"
+                        dataKey="manual/place_addition_input" style="margin: auto">
```

Comment:
don't we want to show this only if place time use is configured?
and shouldn't there be a place_user_input button to match the server side code?

Reply:
This seems like a low priority now, but I agree we should be consistent.

We use `enketo-add-note-button` for both trip-level and place-level notes.
We use `enketo-trip-button` for trip user inputs. I don't think it makes sense to make a new `enketo-place-button`.
Instead we should refactor `enketo-trip-button` into `enketo-userinput-button` or some other generic name

In terms of UI, it is hard for me to imagine where the place userinput button should go, since we don't have a use scenario yet. But for now I could just put whichever buttons are configured on the same row.